doc/user_guide/Large_Timeseries.ipynb

```diff
@@ -242,7 +233,7 @@
    "outputs": [],
    "source": [
     "df0.hvplot(x=\"time\", y=\"value\", rasterize=True, cnorm='eq_hist', padding=(0, 0.1),\n",
-    "           min_height=300, autorange='y', title=\"Datashade\", colorbar=False, line_width=2)"
+    "           min_height=300, autorange='y', title=\"Rasterize\", colorbar=False, line_width=2)"
```

Comment:
Changed the title since `rasterize=True` is set.

Reply:
sounds good, but just curious if there was a reason why you also didn't change the subsequent datashader categories example to `rasterize`? Anyways, I changed it to rasterize, let me know if there's a good reason to use datashade instead.

Reply:
Either I missed it or I wasn't sure what the intention was, anyway, it's now fixed :)